ChatQnA/kubernetes/manifests/docsum_gaudi_llm.yaml

```diff
@@ -0,0 +1,74 @@
+# Source: llm-uservice/charts/tgi/templates/service.yaml
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: docsum-llm-uservice
+  labels:
+    helm.sh/chart: llm-uservice-0.1.0
+    app.kubernetes.io/name: llm-uservice
+    app.kubernetes.io/instance: docsum
+    app.kubernetes.io/version: "1.0.0"
+    app.kubernetes.io/managed-by: Helm
+spec:
+  type: ClusterIP
+  ports:
+    - port: 9000
+      targetPort: 9000
+      protocol: TCP
+      name: llm-uservice
+  selector:
+    app.kubernetes.io/name: llm-uservice
+    app.kubernetes.io/instance: docsum
+---
+# Source: llm-uservice/templates/deployment.yaml
```

Comment:
looks like double license stuff

Reply:
Yeah, I get this yaml file from manifest, this would be split into 2 files, that may be the reason why the license is added in 2 places.

Reply:
thank you!!